index.html

```diff
@@ -118,6 +123,17 @@
             "!WEBIDL#securityerror"><code>SecurityError</code></a>, and abort
             these steps.
             </li>
+            <li>Let <var>url</var> be the result of running the <a data-cite=
```

Comment:
And if so, I think it should be a [TypeError](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror), which is apparently the JavaScript equivalent of Python's ValueError and TypeError combined, not a [SyntaxError](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-syntaxerror) which is reserved for JavaScript parse errors.